src/platform/plugins/shared/dashboard/server/api/search/schemas.ts

```diff
@@ -56,8 +51,16 @@ export const searchResponseBodySchema = schema.object({
         tags: schema.maybe(schema.arrayOf(schema.string())),
         timeRange: schema.maybe(timeRangeSchema),
         title: schema.string(),
+        access_control: schema.maybe(
+          schema.object({
```

Comment:
LEts make accessControlSchema and use here and in dashboard_state_schema to avoid duplication.